lms/static/js/learner_dashboard/views/collection_list_view.js

```diff
@@ -47,6 +47,10 @@
                             this.$el.before(HtmlUtils.ensureHtml(this.getTitleHtml()).toString());
                         }
 
+                        count = this.$el.parent().find('.count');
```

Comment:
Views should not really be updating HTML outside themselves. The View that contains ```.count``` should be listening for a model change or something and update it itself.